build/media-suite_compile.sh

```diff
@@ -1438,6 +1439,11 @@ if [[ $x264 != no ]]; then
         if [[ $standalone = y && $x264 =~ (full|fullv) ]]; then
             _check=("$LOCALDESTDIR"/opt/lightffmpeg/lib/pkgconfig/libav{codec,format}.pc)
             do_vcs "https://git.ffmpeg.org/ffmpeg.git"
+            if [[ $ffmpegVersion == latest ]]; then
+                echo > VERSION
```

Comment:
No, this is also a terrible idea. Why are you forcing FFmpeg to use what's inside this file, which might not even match in anyway the current checked out branch/tag?

Reply:
Do you really verify how it works?

Reply:
Not 100%, but VERSION is used for release branches by FFmpeg, so anyway it would be wrong to write to it.

Reply:
If `echo > VERSION`, FFmpeg won't use the `VERSION` file. So if not do it like this, how do you specify the version shown in `ffmpeg -version`, or do you always build the latest FFmpeg?

Reply:
--extra-version, like I said somewhere else.

Reply:
Yes, you are right.